### PR TITLE
Improvements/tests

### PR DESCRIPTION
**Pull request by lzakrzewski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# CommandBusLauncherBundle
+
+[![Build Status](https://travis-ci.org/Lucaszz/CommandBusLauncherBundle.svg?branch=improvements%2Ftests)](https://travis-ci.org/Lucaszz/CommandBusLauncherBundle)
```
Comment (pawaclawczyk) [inline README.md:3]: Maybe `ClearcodeHQ` should have an account on Travis CI?

File: src/Command/CommandBusLaunchCommand.php [hunk 2]
```diff
@@ -2,44 +2,60 @@
...
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use ClearcodeHQ\CommandBusLauncher\CommandLauncherException;
 
-class CommandLauncherCommand extends ContainerAwareCommand
+class CommandBusLaunchCommand extends ContainerAwareCommand
 {
     const SUCCESS_CODE = 1;
-    const ERROR_CODE   = 0;
+    const ERROR_CODE = 0;
```
Comment (pawaclawczyk) [inline src/Command/CommandBusLaunchCommand.php:14]: Status 0 is usually treat as success exit. http://tldp.org/LDP/abs/html/exit-status.html

File: src/Command/CommandBusLaunchCommand.php [hunk 3]
```diff
@@ -2,44 +2,60 @@
...
 
     protected function configure()
     {
+        //@todo add description
```
Comment (pawaclawczyk) [inline src/Command/CommandBusLaunchCommand.php:18]: You can just add the description ;)

File: src/Command/CommandBusListCommand.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+namespace ClearcodeHQ\CommandBusLauncherBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CommandBusListCommand extends ContainerAwareCommand
+{
+    const SUCCESS_CODE = 1;
+    const ERROR_CODE = 0;
```
Comment (pawaclawczyk) [inline src/Command/CommandBusListCommand.php:12]: As above code 0 is success exit.

File: tests/CLITestCase.php [hunk 5]
```diff
@@ -2,79 +2,74 @@
...
-    protected function executeCommand(Command $command, $parameters = [])
+    protected function assertThatStatusCodeEquals($statusCode)
     {
-        $application = new Application($this->container()->get('kernel'));
-        $application->add($command);
+        $this->assertEquals($statusCode, $this->statusCode);
+    }
 
-        $tester = new CommandTester($command);
+    protected function assertThatOutputWasDisplayed()
```
Comment (pawaclawczyk) [inline tests/CLITestCase.php:61]: Maybe we should add more extended version of this assert like `assertThatOutputContains($string)`.

Comment (lzakrzewski) [inline tests/CLITestCase.php:61]: Why not. But for me assert on exception message is needles.

File: README.md (hunk 1 above)
Comment (lzakrzewski) [inline README.md:3]: I'll try to authirize myself.

Comment (lzakrzewski) [inline README.md:3]: I does not have an access to clearcodehq :/

File: src/Command/CommandBusListCommand.php (hunk 4 above)
Comment (krymen) [inline src/Command/CommandBusListCommand.php:12]: Exit code is needles here as it always returns success.

**Comment by krymen** [comment]
Closed for #2.

**Pull request closed without merging**